packages/openactive-integration-tests/test/features/leasing/named-leasing/implemented/test-opportunity-bookable-test.js

```diff
@@ -0,0 +1,68 @@
+const { expect } = require('chai');
+const { FeatureHelper } = require('../../../../helpers/feature-helper');
+const { GetMatch, C1, C2 } = require('../../../../shared-behaviours');
+const moment = require('moment');
+
+/**
+ * @typedef {import('chakram').ChakramResponse} ChakramResponse
+ */
+
+/**
+ * @param {() => ChakramResponse} getChakramResponse This is wrapped in a
+ *   function because the actual response won't be available until the
+ *   asynchronous before() block has completed.
+ */
+function itShouldReturnLeaseWithFutureExpiryDate(getChakramResponse) {
+  it('should return lease with future expiry date', () => {
+    const chakramResponse = getChakramResponse();
+    const leaseExpires = moment(chakramResponse.body.lease.leaseExpires);
+    expect(leaseExpires.isAfter()).to.be.true;
+  });
+}
+
+FeatureHelper.describeFeature(module, {
+    testCategory: 'leasing',
+    testFeature: 'named-leasing',
+    testFeatureImplemented: true,
+    testIdentifier: 'test-opportunity-bookable',
+    testName: 'Response at C2 includes a "lease" with a "leaseExpires" in the future',
+    testDescription: 'User adds an event to their cart, which reserves it for a specified length of time.',
```

Comment:
we don't have the concept of a "cart", so perhaps:

```suggestion
    testDescription: 'Named lease returned at C2 reserves the OrderItems for a specified length of time',
```